spec/features/manage_tools/open_drafts_spec.rb

```diff
@@ -43,17 +43,43 @@
   context "when more than #{draft_display_max_count} open drafts exist" do
     before do
       # create draft_display_max_count drafts per user
-      draft_display_max_count.times { create(:empty_service_draft, user_id: @current_user_id) }
-      draft_display_max_count.times { create(:empty_service_draft, user_id: @other_user_id) }
+      draft_display_max_count.times { create(:empty_tool_draft, user_id: @current_user_id) }
+      draft_display_max_count.times { create(:empty_tool_draft, user_id: @other_user_id) }
 
       visit manage_tools_path
     end
 
-    # TODO this should be available when the index page is added, MMT_2222
-    it 'the "More" is displayed' # do
-    #   within '.open-drafts' do
-    #     expect(page).to have_link('More')
-    #   end
-    # end
+    it 'the "More" is displayed' do
+      within '.open-drafts' do
+        expect(page).to have_link('More')
+      end
+    end
+
+    context 'when clicking on the "More" link' do
+      before do
+        within '.open-drafts' do
+          click_on 'More'
+        end
+      end
+
+      it 'the Tool Drafts index page is displayed with all open drafts for the provider' do
+        expect(page).to have_content('<Blank Name>', count: (draft_display_max_count * 2))
+      end
+# TODO: add test when index view is supported
```

Comment:
Does this part need to be enabled since index view is being supported?

Reply:
The links that this test is clicking on will provide 404 errors until the ticket Charles is currently working on is finished. I think Charles is expecting to have a PR on Monday for it.  I think the comment should be clarified.